client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -71,20 +71,31 @@ type ChartDataPoint = {
 	value: number;
 };
 
-const transformLineChartData = ( data: SubscribersData[] ): ChartDataPoint[] => {
-	return ( data
-		?.map( ( point ) => {
-			const dateObj = new Date( point.period );
-			if ( isNaN( dateObj.getTime() ) ) {
-				return null;
-			}
+const transformLineChartData = (
+	data: SubscribersData[],
+	hasAddedPaidSubscriptionProduct: boolean
+): ChartDataPoint[][] => {
+	const subscribersData: ChartDataPoint[] = [];
+	const paidSubscribersData: ChartDataPoint[] = [];
+	data?.map( ( point ) => {
+		const dateObj = new Date( point.period );
+		if ( isNaN( dateObj.getTime() ) ) {
+			return null;
+		}
 
-			return {
+		subscribersData.push( {
+			date: dateObj,
+			value: point.subscribers ?? 0,
+		} );
+
+		if ( hasAddedPaidSubscriptionProduct ) {
+			paidSubscribersData.push( {
 				date: dateObj,
-				value: point.subscribers ?? 0,
-			};
-		} )
-		.filter( Boolean ) || [] ) as ChartDataPoint[]; // Type assertion to ensure null values are filtered out
+				value: point.subscribers_paid ?? 0,
```

Comment:
change this line to the following and remove the condition on L91 to be able to see some data:
```
point.subscribers * 0.1;
```